jekyll/_cci2/testing-orbs.md

```diff
@@ -9,177 +9,170 @@ version:
 - Cloud
 ---
 
-This page describes various testing methodologies you can test use to test orbs.
+This guide covers various best practices for testing orbs.
 
 * TOC
 {:toc}
 
 ## Introduction
 
-CircleCI orbs are packages of configuration that you can use in your configurations to simplify your workflows and quickly and easily deploy application and orbs in your workflow. When creating orbs for your workflow, you will want to perform testing to ensure your orb meets your specific needs before deploying and publishing the orb.
+Setting up a solid testing pipeline for your orb is important, as with all software, especially open source, where community contributions are expected. Because orbs are created in YAML, it may seem difficult to test them effectively. With the orb development kit, there is a simple path to implementing a full range of robust tests for your orb.
 
-You can think of orb testing at four levels, in increasing levels of complexity and scope.
+## Validation
 
-* Schema Validation - this can be done with a single CLI command and checks if the orb is well-formed YAML and conforms to the orb schema.
-* Expansion Testing - this can be done by scripting the CircleCI CLI and tests whether the elements of the orb generate the configuration you intended when processing configuration containing those elements.
-* Runtime Testing - this requires setting up separate tests and running them inside a CircleCI build.
-* Integration Testing - this is likely only needed for fairly advanced orbs or orbs designed specifically as public, stable interfaces to 3rd-party services. Doing orb integration tests requires a custom build and your own external testing environments for whatever systems you are integration with.
+The most basic forms of testing for orbs are configuration validation and code linting. When an orb is packed and published it must be both valid YAML, and valid CircleCI syntax. Both of these checks are automatically applied when using the orb development kit, and provided in the project's config file at `.circleci/config.yml`. Config validation and code linting can also be performed manually, locally.
 
-The sections below provide details on each of these techniques for each level of orb testing.
+When you first make a commit to your orb repository, the [test-pack](https://github.com/CircleCI-Public/Orb-Project-Template/blob/43712ad367f2f3b06b2ae46e43ddf70bd3d83222/.circleci/config.yml#L40) workflow will be triggered, which contains several jobs related to validating and testing your orb.
```

Comment:
I know we don't want to include too much content that can get outdated but I think it would be really helpful to add a snippet to show the test-pack workflow in full here. Any objections?